README.md

```diff
@@ -207,10 +207,20 @@ gulp test-coverage
 gulp view-coverage
 ```
 
-For end-to-end testing, edit the example file `./integrationExamples/gpt/pbjs_example_gpt.html`:
+For end-to-end testing:
 
-1. Change `{id}` values appropriately to set up ad units and bidders
-2. Set the path to Prebid.js in your example file as shown below (see `pbs.src`).
+```bash
+gulp e2e-test --host=test.localhost
```

Comment:
Will it work if i use any other alias which is also defined in host file ? e.g my.localhost
If yes can you update the description accordingly.

Reply:
I have that described below?